### PR TITLE
SEC-190 - Consumer loans is incomplete and needs additional content / definitions

### PR DESCRIPTION
**Pull request by ElisaKendall**
## Description

1. Added consumer credit card agreement and commercial credit card agreement to the credit cards ontology; Revised the consumer loans ontology to add a superclass to the new consumer credit card agreement stating that it is a consumer loan
2. Revised the concept of an 'auto loan' to the more general 'motor vehicle loan', added explanatory notes and references and added secured and unsecured consumer loan to account for the legal definitions in US law per discussion in the SEC working group
3. Clarified the definition of consumer loan to explicitly exclude mortgages and student loans as per US law; note that mortgages are included in the broader definition of consumer credit in the UK but student loans are not

Fixes: #1957 / SEC-190

## Checklist:

- [x] I'm familiar with the [FIBO developer quide](https://github.com/edmcouncil/fibo/blob/master/CONTRIBUTING.md#contributing-to-the-fibo-code). My contribution meets all the requirements described there.
- [x] My contribution follows the [principles of best practices for FIBO](https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md).
- [x] My changes have been reconciled with latest master and no merge conflicts remain.
- [x] This PR is related to exactly one issue. The issue is referenced by using a GitHub keyword such as "fixes", "closes", or "resolves".
- [x] Hygiene tests have been applied by a PR with "(WIP)" in title.
- [x] The issue has been tested locally using a reasoner (for ontology changes).

### CODE REVIEW

File: LOAN/LoansSpecific/CardAccounts.rdf [hunk 1]
```diff
@@ -321,6 +363,9 @@
...
+		<cmns-av:explanatoryNote>In its non-physical form, a credit card represents a payment mechanism which facilitates both consumer and commercial business transactions, including purchases and cash advances. A credit card generally operates as a substitute for cash or a check and most often provides an unsecured revolving line of credit. The borrower is required to pay at least part of the card&apos;s outstanding balance each billing cycle, depending on the terms as set forth in the cardholder agreement. As the debt reduces, the available credit increases for accounts in good standing. These complex financial arrangements have ever-shifting terms and prices. A charge card differs from a credit card in that the charge card must be paid in full each month.</cmns-av:explanatoryNote>
+		<cmns-av:explanatoryNote>In physical form, a credit card traditionally is a thin, rectangular plastic card. The front of the card contains a series of numbers that are representative of various items such as the applicable network, bank, and account.</cmns-av:explanatoryNote>
```
Comment (jfgemski) [inline LOAN/LoansSpecific/CardAccounts.rdf:368]: I have at least one credit card where the credit card number is on the back of the card

File: LOAN/LoansSpecific/ConsumerLoans.rdf [hunk 2]
```diff
@@ -64,32 +63,71 @@
...
+		<cmns-av:adaptedFrom xml:lang="en">12 CFR § 228.12, https://www.law.cornell.edu/cfr/text/12/228.12</cmns-av:adaptedFrom>
+		<cmns-av:explanatoryNote xml:lang="en">In the United States, consumer loans do not include a home mortgage, small business, or small farm loans. They are defined by law to include the following: (1) Motor vehicle loan, which is a consumer loan extended for the purchase of and secured by a motor vehicle; (2) Credit card loan, which is a line of credit for household, family, or other personal expenditures that is accessed by a borrower&apos;s use of a &apos;credit card,&apos; as this term is defined in § 1026.2 of this chapter; (3) Other secured consumer loan, which is a secured consumer loan that is not included in one of the other categories of consumer loans; and (4) Other unsecured consumer loan, which is an unsecured consumer loan that is not included in one of the other categories of consumer loans.</cmns-av:explanatoryNote>
```
Comment (jfgemski) [inline LOAN/LoansSpecific/ConsumerLoans.rdf:83]: The reference to "paragraph 1026.2 of this chapter" is unclear as to where that is

File: LOAN/LoansSpecific/ConsumerLoans.rdf [hunk 3]
```diff
@@ -64,32 +63,71 @@
...
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-fbc-dae-dbt;isCollateralizedBy"/>
 				<owl:someValuesFrom rdf:resource="&fibo-fbc-dae-dbt;PhysicalCollateral"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label xml:lang="en">home equity line of credit</rdfs:label>
-		<skos:definition xml:lang="en">line of credit secured by equity value in a borrower&apos;s home or other property</skos:definition>
+		<skos:definition xml:lang="en">line of credit granted to a homeowner secured by the equity value in a borrower&apos;s home or other property</skos:definition>
 		<cmns-av:abbreviation xml:lang="en">HELOC</cmns-av:abbreviation>
```
Comment (jfgemski) [inline LOAN/LoansSpecific/ConsumerLoans.rdf:98]: HELOC -- more an acronym instead of an abbreviation.  Does rdf support acronyms?

**Review by jfgemski — CHANGES_REQUESTED** [review]

File: LOAN/LoansSpecific/ConsumerLoans.rdf (hunk 2 above)
Comment (ElisaKendall) [reply LOAN/LoansSpecific/ConsumerLoans.rdf]: Fixed

**Review by jfgemski — APPROVED** [review]

File: LOAN/LoansSpecific/CardAccounts.rdf (hunk 1 above)
Comment (rivettp) [reply LOAN/LoansSpecific/CardAccounts.rdf:368]: There are also now virtual cards for online payments, for the same account, but with a different card number. Can we model those?

**Review by rivettp — APPROVED** [review]
These changes are OK (though not happy about the jurisdiction-specific of consumer loan) but we need to revisit the CardAccounts ontology as whole since:
 a) I don't understand how to practically use it - there seem to many classes for the same thing
  b) there are some very problematic properties reuses e.g. CreditCardAccount isSignifiedBy CreditCard 
  c) to check complex situations such as corporate card holders and virtual cards

**Review by jfgemski — APPROVED** [review]

**Review by tahoeblue — APPROVED** [review]

**Pull request merged**